shared/src/business/useCases/public/getPublicCaseInteractor.test.ts

```diff
@@ -68,12 +68,12 @@ describe('getPublicCaseInteractor', () => {
       });
   });
 
-  it('should format the given docket number, removing leading zeroes and suffix', async () => {
+  it('should format the given docket number, removing leading zeroes and suffix', () => {
     applicationContext
       .getPersistenceGateway()
       .getCaseByDocketNumber.mockReturnValue(MOCK_CASE);
 
-    await getPublicCaseInteractor(applicationContext, {
+    getPublicCaseInteractor(applicationContext, {
```

Comment:
why was the `await` dropped here?

Reply:
Added the await back in. Maybe an oversight.